contracts/modules/cover/Cover.sol

```diff
@@ -174,6 +174,8 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
       _coverData,
       _coverSegments
     );
+
+    emit CoverMigrated(coverId, fromOwner, toNewOwner, _coverData.length - 1);
```

Comment:
note i'm doing an extra read here.

Reason: returning the cover id from the Cover `CoverUtilsLib.migrateCoverFromOwner` causes stack too deep.

gas_cost = 100 (warm access)